import-service/src/importFileParser.js

```diff
@@ -0,0 +1,45 @@
+const AWS = require("aws-sdk");
+const csv = require("csv-parser");
+const { FOLDER_UPLOAD, FOLDER_PARSED } = require('./configs');
+
+module.exports = async (event) => {
+  console.log('importFileParser input: ',JSON.stringify(event))
+
+  const s3 = new AWS.S3({ signatureVersion: "v4" });
+  const bucketName = event.Records[0].s3.bucket.name;
+  const keyName = event.Records[0].s3.object.key;
+
+  const moveFileToParsed = async () => {
+    await s3
+      .copyObject({
+        Bucket: bucketName,
+        CopySource: bucketName + "/" + event.Records[0].s3.object.key,
+        Key: event.Records[0].s3.object.key.replace(FOLDER_UPLOAD, FOLDER_PARSED),
```

Comment:
how about creating it though iteration of the records instead of using [0] index record everywhere, because if u get multiple records it will throw bug

Reply:
skip this - I found out that it will always be array with one record when s3.objectCreated event is triggerd.